Nightly build files not publishing?

I notice the nightly build doesn't seem to be updating on the MAT update website?

https://download.eclipse.org/mat/snapshots/update-site/

Last publish date looks to be June 6.

  | .. |   |   |  
-- | -- | -- | -- | --
  | binary | 11.4K | 2024-06-06 11:28 |  
  | features | 72K | 2024-06-06 11:28 |  
  | plugins | 15.1M | 2024-06-06 11:28 |  
  | artifacts.jar | 5.6k | 2024-06-06 11:28 |  
  | artifacts.xml.xz | 4.8k | 2024-06-06 11:28 |  
  | content.jar | 24.5k | 2024-06-06 11:27 |  
  | content.xml.xz | 10.9k | 2024-06-06 11:28 |  
  | p2.index | 172 | 2024-06-06 11:28

Do we need to finalise the work in #55 GH Actions workflow? I had assumed the old pipelines would continue to run post-migration.

@krumts it looks like this is dash/cyclone issue,

I can see you made this commit -- 4535fc2, but I do not see exactly where we specify the license. I assume since we do not provide one that it resorts to the dash-license-plugin source, which looks to be 


```
Caused by: java.lang.NullPointerException: Cannot invoke "java.util.Map.keySet()" because "this.approvedLicenses" is null
```

I can see the history here:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=582480

--- a/parent/pom.xml
+++ b/parent/pom.xml
@@ -343,6 +343,9 @@
 				<groupId>org.eclipse.dash</groupId>
 				<artifactId>license-tool-plugin</artifactId>
 				<version>1.0.2</version>
+				<configuration>
+					<licenses>https://www.eclipse.org/legal/licenses/licenses.json</licenses>
+				</configuration>
 				<executions>
 					<execution>
 						<id>license-check</id>

